1Password/op-scim-helm
# Revert redis upgrade

This PR temporarily reverts the redis upgrade and bumps the chart version to v2.4.2.

Note that we no longer are keeping the bridge version (`appVersion` in the `Chart.yaml`) and the Chart version in sync.

To test this we have to ensure that the upgrade from a previous version works as expected as well as test a clean install.

Steps to test:

1. Connect to your k8s cluster of choice (I used DO)
2. Ensure that there is no namespace called `op-scim-bridge`: `kubectl delete namespace op-scim-bridge`
3. Install the previous, working version of the configuration, i.e. v2.3.1, by running this command from the root of the project:
    ```
    helm install op-scim-bridge op-scim-2.3.1.tgz --create-namespace --namespace op-scim-bridge --set scim.credentialsVolume.storageClass=do-block-storage
    ```
4. Ensure that the installation completes successfully and there are no errors in the SCIM bridge logs
5. Package the latest config with: `helm package charts/op-scim-bridge`
6. Upgrade to the latest config:
    ```
    helm upgrade op-scim-bridge op-scim-2.4.2.tgz --create-namespace --namespace op-scim-bridge --set scim.credentialsVolume.storageClass=do-block-storage
    ```
7. Again ensure that the upgrade completes without error and no errors in SCIM bridge logs
8. Uninstall the latest config: `helm uninstall op-scim-bridge --namespace=op-scim-bridge`
9. Do a clean install of the config:
    ```
    helm install op-scim-bridge op-scim-2.4.2.tgz --create-namespace --namespace op-scim-bridge --set scim.credentialsVolume.storageClass=do-block-storage
    ```

Resolves #72.
